multimodal/src/autogluon/multimodal/models/huggingface_text.py

```diff
@@ -125,6 +125,10 @@ def text_segment_ids_key(self):
     def text_valid_length_key(self):
         return f"{self.prefix}_{TEXT_VALID_LENGTH}"
 
+    @property
+    def input_keys(self):
```

Comment:
will this cause backward compatibility issue?

Reply:
1. run_model function would take care of backward compatibility issue.
2. input_key property is not part of model parameter